packages/vuetify/src/components/VChip/_variables.scss

```diff
@@ -2,7 +2,7 @@
 @import '../../styles/tools/_functions.sass';
 
 $chip-avatar-size: 24px !default;
-$chip-close-size: 18px !default;
+$chip-close-size: 24px !default;
```

Comment:
https://material.io/components/chips/#specs

Reply:
everything else is defaulted to 24px which is why there are alignment issues with chip-close... should everything be defaulted to 18? or just the chips?

Reply:
Just chips are 18

Reply:
it okay if i back out the chips and push through the first issue and handle the chips in a separate commit? if not i can do it in this PR just going to be a day or so

Reply:
a day or so is fine

Reply:
updated